### PR TITLE
Update all the things

### PR DESCRIPTION
**Pull request by thebeanogamer**

### CODE REVIEW

**Comment by phantasml** [comment]
Really nice formatting changes, now I can read it without my eyes hurting. Also the clarification on Cyberist/Forensicator is useful because I didn't know what they meant. Good work, thebeanogamer.

File: COMMANDS.md [hunk 1]
```diff
@@ -1,58 +1,51 @@
...
+
+Commands for the community developed bot can be found on the [repository](https://github.com/cyberdiscovery/cyberdisc-bot) for the project.
 
 ## Maths Bot
+
 Command | Purpose
 --- | ---
-!weekly | Responds with the maths question of the week
-!question | Responds with a random maths question
+`!weekly` | Responds with the maths question of the week
```
Comment (teamshortcut) [inline COMMANDS.md]: The prefix is `%`.
```suggestion
`%help` | Lists possible commands and their usage.
`%weekly` | Returns the weekly maths question.
`%question [number] [category]` | Returns a maths question, either random (optionally from category) or with a specific number.
```

**Review by Sh3llcod3 — APPROVED** [review]
*Review all the things*

**Comment by Sh3llcod3** [comment]
Actually, I was going through the diff, when I noticed this duplicate. Also the fact that higher resolution, quality emotes need to be sourced for these, we do need to cater a bit more to Nitro users, they're literally artefacting around the edges (I added them from mobile, so it wasn't evident)
blobs

File: MISCELLANEOUS.md [hunk 2]
```diff
@@ -63,16 +64,118 @@ Name | Picture
...
 --- | ---
 `:blobbongos:` | ![](https://cdn.discordapp.com/emojis/524651455202852864.gif?v=1)
 `:HMMMMMM:` | ![](https://cdn.discordapp.com/emojis/529951282228887562.gif?v=1)
 `:BlobbleWobble:` | ![](https://cdn.discordapp.com/emojis/573579518980390914.gif?v=1)
 `:SuperRotatingThink:` | ![](https://cdn.discordapp.com/emojis/573579519433113610.gif?v=1)
 `:GoogleAssistant:` | ![](https://cdn.discordapp.com/emojis/573579519638634526.gif?v=1)
+`:ablobweary:` | ![](https://cdn.discordapp.com/emojis/665746904348491806.gif?v=1)
+`:ablobsleeping:` | ![](https://cdn.discordapp.com/emojis/665745644878823434.gif?v=1)
+`:ablobquitecool:` | ![](https://cdn.discordapp.com/emojis/665486348999852042.gif?v=1)
+`:ablobwavehello:` | ![](https://cdn.discordapp.com/emojis/665398167260495884.gif?v=1)
```
Comment (Sh3llcod3) [inline MISCELLANEOUS.md]: `:ablobwavehello:`
This one and the one below are the same.

**Review by SSAFuze — APPROVED** [review]

**Pull request merged**